src/thunderbots/software/geom/util.cpp

```diff
@@ -973,6 +999,16 @@ std::pair<Point, Point> getCircleTangentPoints(const Point &start, const Circle
     }
 }
 
+std::pair<Vector, Vector> getCircleTangentVectors(const Point reference,
+                                                  const Circle circle, double buffer)
+{
+    auto [tangent_point1, tangent_point2] =
+        getCircleTangentPoints(reference, circle, buffer);
+
+    return std::make_pair((tangent_point1 - reference).norm(),
```

Comment:
The javadoc comment for this function doesn't mention the returned vectors are normalized. I think it makes the most sense to return the un-normalized vectors, since the caller can normalize them if they want but they can't "un-normalize" them.

Reply:
I've deiced to change the returned values to Rays to better reflect the intention of this function.